aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/TagHelpers/AddressTagHelperComponent.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace RazorPagesSample.TagHelpers
+{
+    #region snippet_AddressTagHelperComponentClass
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Razor.TagHelpers;
+
+    public class AddressTagHelperComponent : ITagHelperComponent
+    {
+        string _printableButton = 
+            "<button type='button' class='btn btn-info' onclick=\"window.open(" +
+            "'https://binged.it/2AXRRYw')\">" +
```

Comment:
Here's an example of an escaped string. No biggie tho.

Reply:
It was done this way to keep under 85 chars per line.

Reply:
Sure ... I was referring to `@` on it with double-quotes "" in the string. Elsewhere, it's that way. Here, it's escaped quotes (\\"). No biggie ... just a small consistency thing. It's not necessarily worth changing. btw--- We never settled as a team on what to do, and I think we're all over the place on this one.